proto/mixing_state.proto

```diff
@@ -17,10 +17,15 @@ message LogSBState {
   Matrix regression_coeffs = 1;
 }
 
+message TruncSBState {
+  Vector sticks = 1;
```

Comment:
I would also save the weights, since it's easier to interpret the weights than the sticks